manifests/g/Git/MinGit/BusyBox/2.43.0/Git.MinGit.BusyBox.installer.yaml

```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: Git.MinGit.BusyBox
+PackageVersion: 2.43.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: cmd\git.exe
+  PortableCommandAlias: git
+- RelativeFilePath: cmd\scalar.exe
+  PortableCommandAlias: scalar

```

Comment:
```suggestion
  PortableCommandAlias: scalar
- RelativeFilePath: mingw64\bin\busybox.exe
  PortableCommandAlias: busybox
```

Reply:
It's basically switching the MinGit runtime from MSYS2 to BusyBox-w32, so I referred to the manifest of Git.MinGit. For more details, please check https://github.com/git-for-windows/git/wiki/MinGit .